DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: rLakeAnalyzer
 Title: Lake Physics Tools
 Maintainer: Luke Winslow <lawinslow@gmail.com>
-Version: 1.10.0
+Version: 1.11.0
 Author: Luke Winslow, Jordan Read, Richard Woolway, Jennifer Brentrup, Taylor
     Leach, Jake Zwart, Sam Albers, Doug Collinge
 Description: Standardized methods for calculating common important derived

```

Comment:
Do we need an expanded `Imports` section to include the more standard packages? `stats`, `graphics`, `utils`, etc. I forget what the current suggestions are now.

Reply:
Also `grDevices`.

Reply:
It is weirdly system specific. I have package [rsoi](https://cran.r-project.org/web/checks/check_results_rsoi.html) where some checks fail and some other don't. I'll change it and see what travis says.

Reply:
Addressed here: https://github.com/boshek/rLakeAnalyzer/commit/4d40ec8d3c6a9e2b759b8ce7ebe5c1ea1e5dc900